src/views/CharacterListView.js

```diff
@@ -10,12 +11,12 @@ class CharacterListView extends React.Component {
   }
 
   componentDidMount() {
-    // call our action
+    this.props.getData()
   }
 
   render() {
     if (this.props.fetching) {
-      // return something here to indicate that you are fetching data
+     <p>LOADING DATA...</p>
```

Comment:
The "p" element needs a *return* keyword.

Reply:
Fixed it.